operator/controllers/keptntask/job_utils.go

```diff
@@ -114,3 +114,12 @@ func (r *KeptnTaskReconciler) getJob(ctx context.Context, jobName string, namesp
 	}
 	return job, nil
 }
+
+func createJobLabels(task klcv1alpha1.KeptnTask) map[string]string {
```

Comment:
can we move this to the common package and use the string constants for the annotation keys?

Reply:
I suggest also generalizing the function name
```suggestion
func createKeptnLabels(task klcv1alpha1.KeptnTask) map[string]string {
```

Reply:
should be done now ...

Reply:
left the function in the controller folder atm as it is specific to tasks ...